.githooks/pre-commit

```diff
@@ -1,8 +1,36 @@
 #!/bin/bash
 cd "$(git rev-parse --show-toplevel)"
+# Pylint Path is Micky's local path, feel free to change in PR 
+PYLINT = "c:\users\micky\appdata\local\programs\python\python310\lib\site-packages\pylint" 
 ESLINT="client/node_modules/.bin/eslint"
 pwd
 
+if [[ ! -x "$PYLINT" ]]; then
+	  printf "\t\033[41mPlease install Pylint\033[0m (pip install pylint)\n"
+	    exit 1
+    fi
+
+    STAGED_FILES=($(git diff --cached --name-only --diff-filter=ACM | grep ".py\{0,1\}$"))
+
+    echo "Checking ${#STAGED_FILES[@]} staged files for Pylint Errors..."
+
+    if [[ "$STAGED_FILES" = "" ]]; then
+	      exit 0
+      fi
+
+      $PYLINT "${STAGED_FILES[@]}" --fix --rcfile /.lint/.pylintrc --load-plugins pylint_django
+
+      PYLINT_EXIT="$?"
+
+      if [[ "${PYLINT_EXIT}" == 0 ]]; then
+	        printf "\n\033[42mPYLINT PASSED, COMMIT SUCCEEDED\033[0m\n"
```

Comment:
Ah yes I think this should just be `printf "\n\033[42mPYLINT PASSED\033[0m\n"` since the commit succeed won't happen until the Eslint check also passes.

Reply:
Oh and I think we can change the message on line 55 to `printf "\n\033[42mESLINT PASSED, COMMIT SUCCEEDED\033[0m\n"`